components_library/processors/cpu_types.py

```diff
@@ -32,3 +32,5 @@ class CPUTypes(Enum):
     KVM = 2
     O3 = 3
     TIMING = 4
+    PYGEN = 5
```

Comment:
This isn't scalable... we're should try to find a better solution here. @BobbyRBruce